data-collection/src/main/java/refactoringml/ProcessMetric.java

```diff
@@ -33,10 +35,11 @@
 	public static String[] bugKeywords = {"bug", "error", "mistake", "fault", "wrong", "fail", "fix"};
 	private Calendar baseCommitDateForNonRefactoring;
 
-	public ProcessMetric (String fileName, String baseCommitForNonRefactoring, Calendar baseCommitDateForNonRefactoring) {
+	public ProcessMetric (String fileName, String baseCommitForNonRefactoring, String baseCommitMessageForNonRefactoring, Calendar baseCommitDateForNonRefactoring) {
```

Comment:
Make this into a class, so that the commit message is stored in a different table. This will facilitate future exports of the dataset (i.e., people that do not need commit messages also do not need to restore this table).